src/instructlab/cli/model/train.py

```diff
@@ -95,7 +95,7 @@ def clickpath_setup(is_dir: bool) -> click.Path:
 @click.option(
     "--local",
     is_flag=True,
-    help="Whether or not `model_dir` is remote from HuggingFace.",
+    help="Whether or not `model_dir` is remote from HuggingFace (use with '--pipeline simple').",
```

Comment:
The description (including before your change) seems to indicate the opposite of what the arg does.

Would this be better as something like:

```Whether or not `model_dir` is local vs remote from Hugging Face (use with '--pipeline simple')```

?

Reply:
Thank you for your suggestion.
And I check again, seems it(`local`) also only use in `MacOS`, should I mention it too?
https://github.com/instructlab/instructlab/blob/main/src/instructlab/model/simple_train.py#L17